instrumentation-annotations/src/test/java/io/opentelemetry/instrumentation/annotations/AddingSpanAttributesUsageExamples.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.annotations;
+
+import io.opentelemetry.api.trace.Span;
+
+/**
+ * This class is not a classical test. It's just a demonstration of possible usages of {@link
+ * AddingSpanAttributes} annotation together with some explanations. The goal of this class is to
+ * serve as an early detection system for inconvenient API and unintended API breakage.
+ */
+@SuppressWarnings("unused")
+public class AddingSpanAttributesUsageExamples {
```

Comment:
The `ExtractAttributesUsingAddingSpanAttributes` already contains an example of usage. WDYT about keeping this in a Javadoc; or even better, adding an example to the [doc page](https://opentelemetry.io/docs/instrumentation/java/automatic/annotations/) once the agent is released?

Reply:
Added this to follow the pattern of the WithSpan annotation.

Agree that the doc page should be update once it is released.

Reply:
Oh, I did not notice the `WithSpanUsageExamples` class. Seems okay then!